src/main/java/com/gooddata/http/client/LoginSSTRetrievalStrategy.java

```diff
@@ -107,7 +104,8 @@ public String obtainSst(final HttpClient httpClient, final HttpHost httpHost) th
                 throw new GoodDataAuthException(message);
             }
 
-            return TokenUtils.extractToken(response);
+            // todo TT is present at response as well - extract it to save one HTTP call
```

Comment:
i don't think we can make such change backward compatible because the strategy needs to be able to return both tokens
So my intention is to leave this for some major rework...